app/frontend/src/router/index.js

```diff
@@ -109,7 +109,7 @@ export default function getRouter(basePath = '/') {
               ),
             meta: {
               breadcrumbTitle: 'Form Designer',
-              requiresAuth: IdentityProviders.IDIR,
+              requiresAuth: true,
```

Comment:
This one should still be IdentityProviders.IDIR as we only want IDIR users to use the form designer

Reply:
Thanks for this. I mixed Form Designer with the Submission Export.

Reply:
The FormDesigner here should still be IDIR

Reply:
This has been fixed